recipe-box-frontend/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM oven/bun:alpine
+
+# Install dependencies
+RUN apk add --no-cache nodejs
+RUN bun i -g serve
+
+# Create app folder and move local files
+RUN mkdir -p /recipe-box
+WORKDIR /recipe-box
+COPY src/main/frontend/package.json /recipe-box/
+COPY src/main/frontend/src/ /recipe-box/src/
+COPY src/main/frontend/.env /recipe-box/
+COPY src/main/frontend/public/ /recipe-box/public/
+COPY src/main/frontend/tsconfig.json /recipe-box/
```

Comment:
Aici ai putea sa ai doua stage-uri in dockerfile, unul de build in care faci npm install si run build, iar un al doilea FROM unde sa copiezi doar artefactul si sa faci serve la el cu nginx sau altceva. In mod normal nu ai tine codul sursa pe container.